E2_ESRGAN/lib/phase_1.py

```diff
@@ -0,0 +1,68 @@
+from functools import partial
+import tensorflow as tf
+from lib import settings, utils, model, dataset
+
+class Model:
```

Comment:
`class Model(object)` for people who might run this with old python. Here and elsewhere.